manifests/version.pp

```diff
@@ -6,37 +6,4 @@
   Optional[String] $scl_httpd_version = undef,
   Optional[String] $scl_php_version   = undef,
 ) {
-  case $facts['os']['family'] {
```

Comment:
Seeing as how everything has been removed should this not ust be deleted?

Reply:
I wonder. It does still have class parameters that can be used. Those could also be added to `params.pp`, but I don't know what's best here.

Reply:
It's a hard question. I guess for now it may be best to leave them in place, at least until hiera is properly implemented within the module. Just so we have a clear place to set them.